docs/docs/ref-01-top-level-api.md

```diff
@@ -90,7 +90,11 @@ Verifies the object is a ReactElement.
 
 ### React.Children
 
-`React.Children` provides utilities for dealing with the `this.props.children` opaque data structure.
+`React.Children` provides utilities for dealing with the `this.props.children` data structure.
```

Comment:
How about:
 > When creating an element using JSX, the type of the `children` prop will often be a string (text), a `ReactElement` (single child node), or an array (multiple children).  Dealing with this union type can be tricky, so React provides a collection of optional helper functions for use with `this.props.children`.

Otherwise, I'm left wondering "what is this mysterious `children` data structure?  Where is that documented?".